schema.json

```diff
@@ -103,10 +122,14 @@
           "type": "string"
         }
       },
+      "required": [
+        "title"
```

Comment:
What's the reasoning behind requiring the title?

Reply:
I think there should be two part to consider for marking which fields should be considered a required field. First is what is the minimum information which is required for rendering a section. For example, in the languages section, a theme can't render the section without the language name. In addition, is the field consider a sensitive field as it could contain some private data.
For the second concern, the schema spilled user's info into the personal and core sections. So everything in the personal section should be optional even personally I believe something like the name is a requirement for a resume.
the title does not considered to be a sensitive field and a theme could use that to render a more meaningful resume so I have marked that as a required field. Although now I'm thinking about it I agree that it is not that much of a requirement. So I will change it.

Reply:
Good reasoning nonetheless. :+1: